lib/pages/home.dart

```diff
@@ -86,7 +86,7 @@ class HomePage extends StatelessWidget {
           package: 'flutter_gallery_assets',
         ),
         assetDarkColor: const Color(0xFF253538),
-        studyRoute: RallyApp.loginRoute,
+        studyRoute: rally_routes.loginRoute,
```

Comment:
Is there a reason why the `Shrine` and `StarterApp` routes aren't changed to do the same as the other studies?

Reply:
Done.